### DISCUSSION TITLE
How is email sending configured?

### DISCUSSION THREAD

**Discussion by ioistired** [Retrospring/retrospring#1009] [Q&A] [answered]
Ok, I have a running instance now, but I don't see how to configure the SMTP server etc to use. So far I have received no email after signing up. How is this configured?

**Comment by nilsding** [comment, marked as answer]
https://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration

**Comment by ioistired** [comment]
Does this go in config/justask.yml?

**Comment by nilsding** [comment]
no.  as the example config below it says

> An example would be adding the following to your appropriate `config/environments/$RAILS_ENV.rb` file

**Comment by ioistired** [comment]
OK, it seems kind of weird to put configs in a file tracked in git but I will put it there.

**Comment by pixeldesu** [comment]
Configuration isn't perfect still, there's #100 to improve on that.

**Comment by greyidol** [comment]
I am working on a PR for this. However, APP_CONFIG is not available in `config/environments/production.rb`. How should I go about this? Should I just use ENV instead?

**Reply by raccube** [reply]
`ENV` is fine.
